app/validators/date_validator.rb

```diff
@@ -19,9 +18,7 @@ def date_of_birth_validations(record, attribute, value)
 
   def date_validations(record, attribute, value)
     return record.errors.add(attribute, :blank_date, article: article(attribute), attribute: humanize(attribute)) if options[:presence] && blank?(value)
-
     return record.errors.add(attribute, :blank_date_fields, attribute: humanize(attribute), fields: blank_fields(value).to_sentence) if !blank?(value) && blank_fields(value).any?
-
```

Comment:
I think this makes it easier to read because the lines are quite busy. If you are going to remove them though, you should also do the same on line 23!